c7n/utils.py

```diff
@@ -25,6 +25,7 @@
 
 from c7n import config
 from c7n.exceptions import ClientError, PolicyValidationError
+from azure.core.exceptions import HttpResponseError
```

Comment:
this should go into c7n_azure utils, rather then creating a hard dependency on azure for the base package. Also note there is separate retry handling there custodian_send_azure_override as an example that tries to incorporate this into the base client.